lfs/credentials.go

```diff
@@ -75,7 +76,7 @@ func getCredURLForAPI(req *http.Request) (*url.URL, error) {
 			if gitRemoteUrl.Scheme == apiUrl.Scheme &&
 				gitRemoteUrl.Host == apiUrl.Host {
 
-				if setRequestAuthFromUrl(req, gitRemoteUrl) {
+				if !Config.NtlmAccess() && setRequestAuthFromUrl(req, gitRemoteUrl) {
```

Comment:
I think these `Config.NtlmAccess()` checks should go inside these functions. This way callers don't need to know to skip `skipCredsCheck()` or `setRequestAuthFromUrl()` if ntlm auth is active.